testrtc.py

```diff
@@ -21,7 +27,45 @@ def get(self, size_kbytes):
     self.response.headers.add_header("Access-Control-Allow-Origin", "*")
     self.response.headers['Content-Type'] = 'application/octet-stream'
     self.response.out.write(random_file[0: int(size_kbytes)*1000])
-    
+
+class NewReportHandler(blobstore_handlers.BlobstoreUploadHandler):
+  # Generate and return blobstore upload URL.
+  def head(self):
+    referer = self.request.headers['Referer']
+    upload_url = blobstore.create_upload_url('/report/new')
+    self.response.headers.add_header("Access-Control-Allow-Origin", referer)
```

Comment:
Indeed I do want to to work on any domain it's hosted on. Is it sufficient to compare the origin header to the referer header? If they differ return 403 or something similar?

Currently it is returning the origin from the request in the Access-Control-Allow-Origin header.